modules/user/user.service.js

```diff
@@ -120,7 +136,16 @@ class UserService {
     );
 
     if (!match) {
-      throw new AuthenticationError('Wrong password');
+      throw new AuthenticationError(`${[
```

Comment:
I think it is not the best idea to put an array inside the template string. Can we pass an array inside `AuthenticationError`? Because if we pass a string it serializes array to something like this:
![image](https://user-images.githubusercontent.com/24845777/87901954-07d10a00-ca61-11ea-97fc-839ae9d42214.png)

Reply:
fixed